src/panel/ThrottleSettings.vue

```diff
@@ -114,11 +114,25 @@ export default defineComponent({
       return val;
     },
     calcThrottle(throttle: number): number {
-      const n = throttle * 2.0 - 1.0;
       const expo = this.profile.rate.throttle_expo;
       const mid = this.profile.rate.throttle_mid;
+      const throttle_minus_mid = throttle - mid;
```

Comment:
can you please guard this new behaviour with `profile.profileVersionGt("0.2.5")` and leave the original in place.
that way it will show the correct graph for older versions.

Reply:
Ah, good point!  I added the original curve logic back in as an `else` against that version check.  I guess it could also be an early return for the new logic and then fall through to this--I'm happy to massage to better blend in to the wider codebase if needed!